src/auto-trace.helper.js

```diff
@@ -84,14 +74,26 @@ export function appendExtraContext(error, extraContext){
 	return errOut
 }
 
+//This function creates an error and by default removes two frames from the error stack (to remove this method, and the caller, which will be a function inside auto-trace)
+//Optionaly extraframesToRemove can be specified to remove more frames if the stack trace will contain more auto-trace refrences.
```

Comment:
nit: typo on optionally
another nit: extraframesToRemove is not camelCased properly